modules/cli/cmd/devrunner/setconfig.go

```diff
@@ -0,0 +1,287 @@
+
+	if updatedRollups == astriaComposerRollups {
+		log.Warn("No changes made to base-config.toml.")
+	}
+
+	if err := config.ReplaceInFile(baseConfigPath, astriaComposerRollups, updatedRollups); err != nil {
+		log.Error("Error updating the file:", baseConfigPath, ":", err)
+		return
+	} else {
+		log.Info("Successfully updated: ", baseConfigPath)
+	}
+	log.Info("Updated 'astria_composer_rollups' to: ", updatedRollups)
+
+	// update networks-config.toml
+	networkConfigs := config.LoadNetworkConfigsOrPanic(networksConfigPath)
+
+	log.Info("Updating rollup_name to", name, "in networks-config.toml for network: ", network)
+	config := networkConfigs.Configs[network]
+	config.RollupName = name
+	networkConfigs.Configs[network] = config
+
+	file, err := os.Create(networksConfigPath)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	// replace the networks config toml with the new config
+	if err := toml.NewEncoder(file).Encode(networkConfigs); err != nil {
+		panic(err)
+	}
+	log.Infof("Successfully updated networks-config.toml: %s\n", networksConfigPath)
+}
+
+// setDefaultDenomCmd represents the set-config default-denom command
+var setDefaultDenomCmd = &cobra.Command{
+	Use:   "default-denom [denom]",
+	Short: "Set the default sequencer denom across all config for the instance.",
+	Args:  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+	Run:   setDefaultDenomCmdHandler,
+}
+
+func setDefaultDenomCmdHandler(c *cobra.Command, args []string) {
+	flagHandler := cmd.CreateCliFlagHandler(c, cmd.EnvPrefix)
+
+	instance := flagHandler.GetValue("instance")
+	config.IsInstanceNameValidOrPanic(instance)
+
+	network := flagHandler.GetValue("network")
+	config.IsInstanceNameValidOrPanic(network)
+
+	baseConfigPath := filepath.Join(cmd.GetUserHomeDirOrPanic(), ".astria", instance, config.DefaultConfigDirName, config.DefaultBaseConfigName)
+	networksConfigPath := filepath.Join(cmd.GetUserHomeDirOrPanic(), ".astria", instance, config.DefaultNetworksConfigName)
+	cometbftGenesisPath := filepath.Join(cmd.GetUserHomeDirOrPanic(), ".astria", instance, config.DefaultConfigDirName, config.DefaultCometbftGenesisFilename)
+
+	denom := args[0]
+	denom = strings.ToLower(denom)
+	config.IsValidDenomOrPanic(denom)
+
+	// Update base-config.toml
+	baseConfig := config.LoadBaseConfigOrPanic(baseConfigPath)
+	astriaComposerFeeAsset := baseConfig["astria_composer_fee_asset"]
+
+	if err := config.ReplaceInFile(baseConfigPath, astriaComposerFeeAsset, denom); err != nil {
+		log.Error("Error updating the file:", baseConfigPath, ":", err)
+		return
+	} else {
+		log.Info("Successfully updated: ", baseConfigPath)
+	}
+	log.Info("Updated 'astria_composer_fee_asset' to: ", denom)
+
+	// Update networks-config.toml
+	networkConfigs := config.LoadNetworkConfigsOrPanic(networksConfigPath)
+
+	log.Info("Updating default_denom to", denom, "in networks-config.toml for network: ", network)
+	config := networkConfigs.Configs[network]
```

Comment:
This collides with the `config` package we use in this file. This occurs in a few other places too, but I haven't commented on them all. Make sure you get them all.